source/ClientRequest/AMDSClientContinuousDataRequest.cpp

```diff
@@ -46,6 +55,9 @@ AMDSClientContinuousDataRequest& AMDSClientContinuousDataRequest::operator =(con
 	if(this != &other){
 		AMDSClientDataRequest::operator =(other);
 
+		setBufferNames(other.bufferNames());
```

Comment:
What is the difference between these two?  Is one the whole list of buffer names and the other is an explicit buffer name for some sort of exclusive mode?